api/src/utils/get-default-value.ts

```diff
@@ -1,12 +1,23 @@
-import { SchemaOverview } from '@directus/schema/dist/types/overview';
```

Comment:
It seems to be possible to resolve this by adding this to the `exports` field of `@directus/schema`'s package.json file:

```js
		"./dist/types/overview": {
			"types": "./dist/types/overview.d.ts",
			"default": "./dist/types/overview.js"
		},
```
which feels like might be better than redeclaring the type here. However admittedly I'm not 100% sure about this solution (as it's just roughly based on what I've looked up from https://github.com/vitejs/vite/issues/10481#issuecomment-1280335703) and there might be better ways to go about it, so I'll have to defer to @nickrum for this as well 😄

Reply:
Good one! After fighting typescript for hours I couldn't be bothered with figuring out more exports nightmares hahah 👉🏻👈🏻

Reply:
Wouldn't it be a bit nicer like this?:
```		
		"./types/overview": {
			"types": "./dist/types/overview.d.ts",
			"default": "./dist/types/overview.js"
		},
```

Import would need to be changed to this:
```diff
- import { SchemaOverview } from '@directus/schema/dist/types/overview';
+ import { SchemaOverview } from '@directus/schema/types/overview';
```

(Would correspond to `exports` in [`@directus/shared`](https://github.com/directus/directus/blob/main/packages/shared/package.json))